src/main/java/no/unit/nva/model/contexttypes/LinkedContext.java

```diff
@@ -0,0 +1,10 @@
+package no.unit.nva.model.contexttypes;
+
+import java.net.URI;
+
+public interface LinkedContext extends PublicationContext {
```

Comment:
This kind of explains what is the difference between the `LinkedContext` and the `BaseContext`. Although I kind of understand what is the difference between the `BaseContext`  and what the `LinkedContext`, it not tottaly clear. A documentation text (or link) would be helpful